lib/api/bucketDeleteWebsite.js

```diff
@@ -0,0 +1,43 @@
+import { errors } from 'arsenal';
+
+import bucketShield from './apiUtils/bucket/bucketShield';
+import { isBucketAuthorized } from './apiUtils/authorization/aclChecks';
+import metadata from '../metadata/wrapper';
+
+const requestType = 'bucketOwnerAction';
+
+export default function bucketDeleteWebsite(authInfo, request, log, callback) {
+    const bucketName = request.bucketName;
+    const canonicalID = authInfo.getCanonicalID();
+
+    return metadata.getBucket(bucketName, log, (err, bucket) => {
+        if (err) {
+            log.debug('metadata getbucket failed', { error: err });
+            return callback(err);
+        }
+        if (bucketShield(bucket, requestType)) {
+            return callback(errors.NoSuchBucket);
```

Comment:
will you please add a test here:
https://github.com/scality/S3/blob/master/tests/unit/api/transientBucket.js
and here:
https://github.com/scality/S3/blob/master/tests/unit/api/deletedFlagBucket.js

to show that this api is shielded?